### PR TITLE
Made a Card Design

### PR DESCRIPTION
**Pull request by harshhgithub**
Hi , made a card design

### CODE REVIEW

**Comment by harshhgithub** [comment]
Hey , i made the card design

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
@AshishV306 

Hello and hope you are doing great?

Thank you for contributing. Before I accept, I need you to make some few changes.
No need for the head section as this module is suppose to be easily reusable by other designers. Please remove the head section and the body tag. we only need three(or less than 3 tags) that includes: "div", "style", "script" tag. You can check the accepted pull request for button designs. (File: card_1.html)

Thank you. You can drop a message whenever you need more info. And by the way this is a good job.

**Comment by harshhgithub** [comment]
hi i made the changes please check now

**Comment by harshhgithub** [comment]
hi made the changes for the pull request #286 . Thanks

File: modules/cards/card_1.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        </div>
+    </div>
+    <style>
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins',sans-serif;
+}
+body{
```
Comment (NtemKenyor) [inline modules/cards/card_1.html]: This style would affect the entire project.

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
**Missed something - One Last Change would do**

Hello and hope you are doing great. 


Avoid styling the general tags(body, head, html etc) or (every tag at once) as this would affect the entire project. Don't style or add scripts to anything outside your parent/first "div".  The goal is to make a module that is simple, dynamic and easily reusable.

card_NUM_(YOUR_NAME).html
```
<!---- Your styling would enter inside the style tag ----->
<style>

</style>

<!---- All scripts would enter inside the script tag ----->
<script>

</script>
<!---- Normal HTML Flow ----->
<div>
    <div class="center-div">
    ...
    </div>
</div>
```

Thank you.

File: modules/cards/card_1.html (hunk 1 above)
Comment (harshhgithub) [reply modules/cards/card_1.html]: okay . I'll make the changes

**Comment by harshhgithub** [comment]
hello , i made the changes to the pr #286 please check

**Review by NtemKenyor — APPROVED** [review]
Hey @harshhgithub 

Big shoutout for your awesome contribution to open source! Your one pull request might be small in number, but it's huge in impact. Thanks for adding your unique touch to the community. Keep the open source spirit alive!

Cheers,

**Pull request merged**